src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -38,32 +41,18 @@ def update
       format.js do
         render partial: 'update', locals: {
           notifications: paginated_notifications,
-          selected_filter: selected_filter,
+          selected_filter: @selected_filter,
           show_read_all_button: show_read_all_button?,
-          user: User.session
+          user: @current_user
```

Comment:
Why do you do this? Everybody knows what `User.session` is, everybody has to look up what `@current_user` is...